src/sdl/SDL/SDL.spec.ts

```diff
@@ -0,0 +1,109 @@
+import template from "lodash/template";
+
+import { readYml } from "../../../test/read-yml";
+import { SDL } from "./SDL";
+import { CustomValidationError } from "../../error";
+import { faker } from "@faker-js/faker";
+import { v2ServiceImageCredentials } from "../types";
+import { omit } from "lodash";
+
+const createYML = template(readYml("sdl-hello-world-basic-with-creds"));
```

Comment:
Interesting use of `template`, we could use this method to simply some of the other tests eventually and reduce the number of SDL templates.

Reply:
after thinking a bit I improved it further https://github.com/akash-network/akashjs/pull/82/commits/56059cbfb5241a9678dbec04aec7e322709b2626